plugins/Login/lang/en.json

```diff
@@ -1,68 +1,68 @@
+        "LoginOrEmail": "Username or e-mail",
+        "HelpIpRange": "Enter one IP address or one IP range per line. You can use CIDR notation e.g: %1$s or you can use wildcards, e.g: %2$s or %3$s",
+        "SettingBruteForceEnable": "Brute-force Detection",
+        "SettingBruteForceEnableHelp": "Logs out users making too many password guesses within a timeframe for a while. This prevent anyone from testing testing all combinations. Getting a shared IP blocked also locks out its other users.",
         "SettingBruteForceWhitelistIp": "Never block these IPs from logging in",
         "SettingBruteForceBlacklistIp": "Always block these IPs from logging in",
-        "NotAllowListTakesPrecendence": "If an IP matches both the block and never block list, then the block list takes precendence and the IP would not be allowed.",
-        "SettingBruteForceMaxFailedLogins": "Number of allowed login retries within time range",
-        "SettingBruteForceMaxFailedLoginsHelp": "If more than this number of failed logins are recorded within the time range configured below, block the IP.",
+        "NotAllowListTakesPrecendence": "If an IP is on the both the list of IPs block and never block it is blocked.",
+        "SettingBruteForceMaxFailedLogins": "Number of allowed login retries within a given time frame",
+        "SettingBruteForceMaxFailedLoginsHelp": "Blocks the IP if more than this number of failed logins are recorded within the time frame set below.",
         "SettingBruteForceTimeRange": "Count login retries within this time range in minutes",
-        "SettingBruteForceTimeRangeHelp": "Enter a number in minutes.",
-        "ConfirmPasswordReset": "Confirm password reset",
-        "ConfirmPasswordResetIntro": "To confirm it is really you who requested this password change, please enter your new password again.",
-        "ConfirmPasswordResetWrongPassword": "The entered password does not match your new password. If you don't remember your newly chosen password you can reset your password again. If you didn't request the password change, simply do nothing and your password won't be changed.",
-        "LoginNotAllowedBecauseBlocked": "You are currently not allowed to log in because you had too many failed logins, try again later.",
-        "CurrentlyBlockedIPs": "Currently blocked IPs",
+        "SettingBruteForceTimeRangeHelp": "Enter a number of minutes.",
+        "ConfirmPasswordReset": "Confirm resetting your password",
```

Comment:
```suggestion
        "ConfirmPasswordReset": "Resetting password…",
```
?

Reply:
This one is used as headline when resetting the password
![image](https://user-images.githubusercontent.com/1579355/173038916-d27ffd12-3b2e-43e2-a392-9c123447faed.png)

Reply:
I left it as my suggestion was clunky. I think "Reset password" works much better, since it avoids the re-explanation.